algos/src/sample_rate_conversion.rst

```diff
@@ -75,139 +75,138 @@ scripts print intermediate information about progress).
 Basic usage
 ===========
 
-First an Octave shell is launched from the tool directory:
+First, an Octave shell is launched from the tool directory:
 
 .. code-block:: bash
 
 		cd tools/tune/src
 		octave --gui
-		
-SRC component is set up with script src_generate.m. A help for
-script usage is printed with Octave shell command:
+
+The SRC component is set up in the src_generate.m script. A help for
+script usage can be printed by using the Octave shell command:
 
 .. code-block:: octave
 
    >> help src_generate
 
-To create SRC coefficients for input rates 32, 44.1, and 48 kHz and
-output rates 44.1 and 48 kHz command would be
+SRC coefficients for input rates 32, 44.1, and 48 kHz and output rates 44.1
```

Comment:
sorry a mistake,"input rates 32, 44.1, and 48 kHz" -> "input rates 32 and 48 kHz"